SharpRepository.Repository/RepositoryBase.cs

```diff
@@ -33,7 +33,7 @@ protected RepositoryBase(ICachingStrategy<T, TKey> cachingStrategy = null)
             _aspects = entityType.GetAllAttributes<RepositoryActionBaseAttribute>(inherit: true)
                 .Where(x => x.Enabled)
```

Comment:
Thank you in advance for accepting my changes. Any ideas when they are available via nuget?